src/meltano/cli/cli.py

```diff
@@ -123,3 +118,38 @@ def cli(  # noqa: WPS231
             "For more details, visit https://docs.meltano.com/guide/installation#upgrading-meltano-version"
         )
         sys.exit(3)
+
+
+def activate_environment(ctx: click.Context, project: Project) -> None:
+    """Activate the selected environment.
```

Comment:
```suggestion
    """Activate the selected environment.

    The selected environment is whatever was selected with the `--environment`
    option, or the default environment (set in `meltano.yml`) otherwise.
```